doc/user-stories/expert-partitioner.md

```diff
@@ -417,3 +420,148 @@ The specs are based on current version of expert partitioner for TW. The goal is
+* shows a table with the columns: Device, Size, F, Enc, Type, FS Type, Label, Mount Point, RAID Type, Chunk Size
+  * and the table is filled out with info of all RAID devices
+* *(pending) shows the actions: 'Add RAID', 'Edit', 'Resize' and 'Delete'*
+* *(pending) and action 'Add RAID' is selected*
+  * and there is less than two not mounted partitions
+    * shows an error message
+  * and there is at least two not mounted partitions with correct partition ID (LINUX, SWAP, LVM, RAID)
+    * shows a wizard to create a new RAID
+* *(pending) and action 'Edit' is selected*
+  * shows dialog to edit the RAID partition
+* *(pending) and action 'Resize' is selected*
+  * shows a dialog to modify the RAID devices
+* *(pending) and action 'Delete' is selected*
+  * shows a confirm popup to delete the RAID
+  * deletes the RAID
+
+### *(pending) When a 'RAID' is selected in the tree view*
+* shows a view with two tabs: 'Overview' and 'Used Devices'
+* and tab 'Overview' is selected
+  * shows a report with three sections: 'Device', 'RAID' and 'File System'
+    * where 'Device' section contains the folling info
+      * device name
+      * device size
+      * encrypted
+      * device id
+    * where 'RAID' section contains the following info
+      * RAID type
+      * chunk size
+      * parity algorithm
+    * where 'File System' section contains the following info
+      * file system
+      * mount point
+      * label
+  * shows the actions: 'Edit', 'Resize' and 'Delete'
+    * and action 'Edit' is selected
+      * shows dialog to edit the RAID partition (similar to edit disk partition)
+    * and action 'Resize' is selected
+      * shows a dialog to modify the RADID devices
+    * and action 'Delete' is selected
+      * shows a confirm popup to delete the RAID
+      * deletes the RAID
+* and tab 'Used Devices' is selected
+  * shows a table with the columns: Device, Size, F, Enc and Type
+    * and the table is filled out with info of all devices that belong to the RAID
+  * and an item in the table is selected (double click)
+    * jumps to the corresponding partition in the 'Hard Disks' section
+
+### *(pending) When 'Add RAID' is selected*
+* shows a wizard with 5 steps
+  * select RAID type and devices
+  * select RAID options
+  * select partition role
+    * same wizard step than in partition creation (see above)
+  * set partition attributes (fs, mount point, etc)
+    * same wizard step than in partition creation (see above)
+    * 'File System Id' is not shown (automatically set to Linux RAID)
+  * set encrypt password (optional)
+    * same wizard step than in partition creation (see above)
+
+#### *(pending) When we are in wizard step to select RAID type and devices*
+* shows the following options
+  * RAID Type
+    * RAID 0 (at least 2 devices)
+    * RAID 1 (at least 2 devices)
+    * RAID 5 (at least 3 devices)
+    * RAID 6 (at least 4 devices)
+    * RAID 10 (at least 2 devices)
```

Comment:
Don't we need at least 4 devices for RAID 10?

Reply:
That's what I though when I saw that line. So I tested it and the expert partitioner allows you to continue with 2.